aspnetcore/fundamentals/error-handling.md

```diff
@@ -60,39 +60,44 @@ public IActionResult Index()
 
 ## Configuring status code pages
 
-By default, your app won't provide a rich status code page for HTTP status codes such as 500 (Internal Server Error) or 404 (Not Found). You can configure the `StatusCodePagesMiddleware` by adding a line to the `Configure` method:
+By default, an app doesn't provide a rich status code page for HTTP status codes, such as *404 Not Found*. To provide status code pages, configure the Status Code Pages Middleware by adding a line to the `Configure` method:
 
 ```csharp
 app.UseStatusCodePages();
 ```
 
-By default, this middleware adds simple, text-only handlers for common status codes, such as 404:
+By default, Status Code Pages Middleware adds simple, text-only handlers for common status codes, such as 404:
 
 ![404 page](error-handling/_static/default-404-status-code.png)
 
-The middleware supports several different extension methods. One takes a lambda expression, another takes a content type and format string.
+The middleware supports several extension methods. One method takes a lambda expression:
 
 [!code-csharp[](error-handling/sample/Startup.cs?name=snippet_StatusCodePages)]
 
+Another method takes a content type and format string:
+
 ```csharp
 app.UseStatusCodePages("text/plain", "Status code page, status code: {0}");
 ```
 
-There are also redirect extension methods. One sends a 302 status code to the client, and one returns the original status code to the client but also executes the handler for the redirect URL.
+There are also redirect and re-execute extension methods. The redirect method sends a 302 status code to the client:
 
 [!code-csharp[](error-handling/sample/Startup.cs?name=snippet_StatusCodePagesWithRedirect)]
 
+The re-execute method returns the original status code to the client but also executes the handler for the redirect URL:
+
 ```csharp
 app.UseStatusCodePagesWithReExecute("/error/{0}");
 ```
 
-If you need to disable status code pages for certain requests, you can do so:
+To disable status code pages for certain requests, disable the feature:
```

Comment:
Does this code have to execute anywhere specific (e.g., in Startup)? If so, it's worth specifying that detail here.